crates/red_knot/src/types.rs

```diff
@@ -37,40 +38,137 @@ impl TypeEnvironment {
     }
 }
 
+#[derive(Debug)]
 pub(crate) enum Type {
-    Class(ClassType),
+    // a specific function
     Function(FunctionType),
+    // the set of Python objects with this class in their MRO
```

Comment:
What's an MRO? Can we add a link or spell the word out?

Reply:
method resolution order -- the linearized list of base classes for a class. I'll spell it out.